### PR TITLE
CI Fix

### PR DESCRIPTION
**Pull request by nabobalis**
Aim to release an update.

### CODE REVIEW

File: sunkit_image/time_lag.py [hunk 1]
```diff
@@ -138,8 +137,17 @@ def _dask_check(signal, lags):
         return lags
     if isinstance(signal, dask.array.Array):
         return dask.array.from_array(lags, chunks=lags.shape)
-    else:
-        return lags
+    return lags
+
+
+def _deal_with_dask(array):
```
Comment (nabobalis) [inline sunkit_image/time_lag.py:143]: A hack.

Comment (wtbarnes) [reply sunkit_image/time_lag.py:143]: I am very very 👎  on this. This defeats the purpose of using putting a Dask array in here at all.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,3 +1,11 @@
+0.3.1 (2021-11-18)
+==================
+
+Bug Fixes
+---------
+
+- Fixed issue with ``time_lag`` and dask arrays.
```
Comment (wtbarnes) [inline CHANGELOG.rst:7]: What is the issue?

File: sunkit_image/time_lag.py [hunk 3]
```diff
@@ -213,6 +221,7 @@ def time_lag(signal_a, signal_b, time: u.s, lag_bounds: (u.s, None) = None, **kw
     lags = get_lags(time)
     cc = cross_correlation(signal_a, signal_b, lags)
     start, stop = _get_bounds_indices(lags, lag_bounds)
+    cc = _deal_with_dask(cc)
```
Comment (wtbarnes) [inline sunkit_image/time_lag.py:224]: See above

**Review by wtbarnes — DISMISSED** [review]

File: CHANGELOG.rst (hunk 2 above)
Comment (nabobalis) [reply CHANGELOG.rst:7]: See https://dev.azure.com/sunpy/sunkit-image/_build/results?buildId=21411&view=logs&jobId=0609049e-3a2a-5217-995e-2cc283dcaa52&j=0609049e-3a2a-5217-995e-2cc283dcaa52&t=8711ded8-03e2-5951-4fd3-69b4a92338f7

**Pull request merged**